packages/react-client/src/ReactFlightClient.js

```diff
@@ -108,6 +118,8 @@ function readChunk<T>(chunk: SomeChunk<T>): T {
       return chunk._value;
     case RESOLVED_MODEL:
       return initializeModelChunk(chunk);
+    case RESOLVED_PROVIDER:
+      return initializeProviderChunk(chunk);
```

Comment:
Note how we don't have one for Symbols. You can remove this an instead always eagerly initialize it.